Python/instrumentation.c

```diff
@@ -915,12 +927,7 @@ get_tools_for_instruction(PyCodeObject *code, PyInterpreterState *interp, int i,
         }
     }
     else {
-        if (code->_co_monitoring) {
-            tools = code->_co_monitoring->active_monitors.tools[event];
-        }
```

Comment:
Because if `!PY_MONITORING_IS_INSTRUMENTED_EVENT(event)` then `event` would be out of bounds.
IIRC this was causing the crash in https://github.com/python/cpython/issues/108390